src/plugins/vis_default_editor/public/components/sidebar/sidebar_title.tsx

```diff
@@ -100,7 +100,7 @@ export function LinkedSearch({ savedSearch, eventEmitter }: LinkedSearchProps) {
       </EuiFlexItem>
 
       <EuiFlexItem grow={false} className="eui-textTruncate">
-        <EuiTitle size="xs" className="eui-textTruncate">
+        <EuiTitle size="m" className="eui-textTruncate">
```

Comment:
Let's not change these xs ones - if they're using that, they probably intentionally need to be very small.